### PR TITLE
#166265842 sign-up validation

### PR DESCRIPTION
**Pull request by PelumiAlesh**
### What does this PR do?
Add validation to the sign-up endpoint
#### Description of Task to be completed?
When a user tries to sign up with the wrong or unsupported input format, the user should receive descriptive signup/Registration validation error(s)
#### Any background context you want to provide?
No
#### What are the relevant pivotal tracker stories?
[#166265842](https://www.pivotaltracker.com/story/show/166265842)
#### Screenshots (if appropriate)
Screenshot 2019-05-31 at 11 41 25 AM
#### Questions:

### CODE REVIEW

**Review by ocranbillions — DISMISSED** [review]

File: server/__test__/test.spec.js [hunk 1]
```diff
@@ -53,4 +53,28 @@ describe('test for user signup', () => {
...
   });
-});
\ No newline at end of file
+});
+
+describe('test for user signup Validation', () => {
+  beforeEach(async () => {
+       await truncate();
+      });
+  it('Should reigister a user when all required input is supplied', async (done) => {
```
Comment (chuxmykel) [inline server/__test__/test.spec.js:85]: I'm sure this isn't supposed to be the title of this `it` block.

**Review by chuxmykel — DISMISSED** [review]
Check the title of that it block and you're good to go

**Review by Hadeneekeh — DISMISSED** [review]

**Review by ocranbillions — DISMISSED** [review]
Handle the merge conflicts and its good to go.

**Review by chuxmykel — DISMISSED** [review]
Work on merge conflict and you're good to go.

**Review by encodedBicoding — COMMENTED** [review]
Gone through the work, it is good. Just needs to fix merge conflicts.

**Review by OvieMudi — DISMISSED** [review]

**Review by encodedBicoding — CHANGES_REQUESTED** [review]
Why is travis failing?

**Review by encodedBicoding — COMMENTED** [review]
that sequelize table issh is causing build to fail. Are you using postgresDB or elephant?.

**Comment by PelumiAlesh** [comment]
> that sequelize table issh is causing build to fail. Are you using postgresDB or elephant?.

Im using postgresDB

**Comment by emp-daisy** [comment]
Why is this still pending?

**Comment by PelumiAlesh** [comment]
THIS IS THE OLD BRANCH

**Pull request closed without merging**